project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
> What are the stability guarantees of 0.13.16-M1?

What is meant here by stability?

> Why was that version not published as 0.13.16?

So we could start using the cross-building functionality, without a full release.

> When can we expect 0.13.16 final?

I'll have a chat with @eed3si9n today. We need to get a version out that includes https://github.com/sbt/sbt/pull/3329 (to avoid everyone having to include the workaround), but Eugene said he had an interest in trying to get https://github.com/sbt/sbt/issues/3205 into 0.13.16 too.

Notes:

+ https://github.com/sbt/sbt/compare/v0.13.15...v0.13.16-M1
+ https://github.com/sbt/sbt/compare/v0.13.16-M1...0.13